data/events.json

```diff
@@ -1419,5 +1419,138 @@
     "end": "2023-07-12 12:00",
     "groupId": 1,
     "filterId": 124
+  },
+  {
+    "title": "[UMinho] Renovação de Inscrição",
+    "start": "2023-09-05 00:00",
+    "end": "2023-09-08 23:59",
+    "groupId": 6,
+    "filterId": 1
+  },
+  {
+    "title": "[UMinho] Início de Semestre (1º)",
+    "start": "2023-09-11 00:00",
+    "end": "2023-09-11 00:00",
+    "groupId": 6,
+    "filterId": 1
+  },
+  {
+    "title": "[UMinho] Férias Natal",
+    "start": "2023-12-18 00:00",
+    "end": "2023-12-30 23:59",
+    "groupId": 6,
+    "filterId": 1
+  },
+  {
+    "title": "[UMinho] Fim de Semestre (1º)",
+    "start": "2024-02-03 00:00",
+    "end": "2024-02-03 00:00",
+    "groupId": 6,
+    "filterId": 1
+  },
+  {
+    "title": "[UMinho] Início de Semestre (2º)",
+    "start": "2024-02-05 00:00",
+    "end": "2024-02-05 00:00",
+    "groupId": 6,
+    "filterId": 1
+  },
+  {
+    "title": "[UMinho] Férias Páscoa",
+    "start": "2024-03-25 00:00",
+    "end": "2024-04-01 23:59",
+    "groupId": 6,
+    "filterId": 1
+  },
+  {
+    "title": "[UMinho] Fim de Semestre (2º)",
+    "start": "2024-06-22 00:00",
+    "end": "2024-06-22 00:00",
+    "groupId": 6,
+    "filterId": 1
+  },
+  {
+    "title": "[UMinho] Lançamento de Pautas (1º Sem)",
+    "start": "2024-02-16 00:00",
+    "end": "2024-02-16 00:00",
+    "groupId": 6,
+    "filterId": 1
+  },
+  {
+    "title": "[UMinho] Lançamento de Pautas (2º Sem)",
+    "start": "2024-06-27 00:00",
+    "end": "2024-06-27 00:00",
+    "groupId": 6,
+    "filterId": 1
+  },
+  {
+    "title": "[UMinho] Feriado",
```

Comment:
What about adding a "Portugal" category? 👀

Reply:
It would make sense really, but I think it also makes sense for it to be UMinho since they are the feriados (xD) that will be respected by the university. Also I'm afraid that with too many filters people will miss out on that information, or we should set some filters to be always active.

Reply:
I think those type of things like an holiday , shouldn't have a filter associated to and always appear.
What your thoughts on that?

Reply:
I think it makes sense, @MarioRodrigues10.

Reply:
@RuiL1904 @MarioRodrigues10 should I make UMinho events always active and with no filter?